files/__addonLocation__/package.json

```diff
@@ -67,20 +67,21 @@
     "@types/ember__modifier": "^4.0.8",
     "@types/ember__helper": "^4.0.5",
     "@types/ember__routing": "^4.0.19",
-    "@typescript-eslint/eslint-plugin": "^6.14.0",
-    "@typescript-eslint/parser": "^6.14.0",<% } %>
+    "@typescript-eslint/eslint-plugin": "^7.7.1",
+    "@typescript-eslint/parser": "^7.7.1",<% } %>
     "@rollup/plugin-babel": "^6.0.4",
     "babel-plugin-ember-template-compilation": "^2.2.2",
     "concurrently": "^8.2.2",
-    "ember-template-lint": "^5.13.0",
-    "eslint": "^8.56.0",
+    "ember-template-lint": "^6.0.0",<% if (packageManager === 'npm') { %>
+    "ember-eslint-parser": "^0.4.2",
```

Comment:
this bit here is sorta cheeky, but npm is the only package manager that somehow is preventing eslint-plugin-ember from resolving one of its own dependencies :thinking: